src/amo/components/WrongPlatformWarning/index.js

```diff
@@ -5,45 +5,39 @@ import * as React from 'react';
 import { compose } from 'redux';
 import { connect } from 'react-redux';
 
-import {
-  CLIENT_APP_ANDROID,
-  INCOMPATIBLE_ANDROID_UNSUPPORTED,
-  INCOMPATIBLE_FIREFOX_FOR_IOS,
-} from 'core/constants';
+import { MOBILE_HOME_PAGE_LINK } from 'core/constants';
 import translate from 'core/i18n/translate';
 import { sanitizeHTML } from 'core/utils';
 import {
   correctedLocationForPlatform,
-  getClientCompatibility,
-  isFenix,
+  isFenixCompatible,
+  isFirefoxForAndroid,
+  isFirefoxForIOS,
 } from 'core/utils/compatibility';
 import Notice, { warningInfoType } from 'ui/components/Notice';
 import type { AppState } from 'amo/store';
 import type { AddonType } from 'core/types/addons';
 import type { UserAgentInfoType } from 'core/reducers/api';
-import type { AddonVersionType } from 'core/reducers/versions';
 import type { I18nType } from 'core/types/i18n';
 import type { ReactRouterLocationType } from 'core/types/router';
 
 import './styles.scss';
 
-export const FENIX_LINK_DESTINATION =
+export const ANDROID_SUMO_LINK_DESTINATION =
   'https://support.mozilla.org/kb/find-and-install-add-ons-firefox-android';
 
 type Props = {|
   addon?: AddonType | null,
   className?: string,
-  currentVersion?: AddonVersionType | null,
-  fixAndroidLinkMessage?: string,
-  fixFirefoxLinkMessage?: string,
-  fixFenixLinkMessage?: string,
+  isHomePage?: boolean,
 |};
 
 type InternalProps = {|
   ...Props,
   _correctedLocationForPlatform: typeof correctedLocationForPlatform,
-  _getClientCompatibility: typeof getClientCompatibility,
-  _isFenix: typeof isFenix,
+  _isFenixCompatible: typeof isFenixCompatible,
```

Comment:
Should it be "Fenix compatible" or something more like "add-on install-able Android"?

Reply:
Yeah, we shouldn't need to reference Fenix anymore, as it's for all Android. I guess `isAndroidInstallable`.